opencti-platform/opencti-graphql/src/python/testing/local_exporter.py

```diff
@@ -6,18 +6,19 @@
 
 
 class TestLocalExporter:
-    def __init__(self, api_url, api_token, entity_id, file_name, file_markings):
+    def __init__(self, api_url, api_token, entity_id, entity_type, file_name, file_markings):
         self.api_url = api_url
         self.api_token = api_token
         self.entity_id = entity_id
+        self.entity_type = entity_type
         self.file_name = file_name
         self.file_markings = file_markings
 
     def upload(self):
         opencti_api_client = OpenCTIApiClient(self.api_url, self.api_token)
         # Generate a json bundle from openCTI
         bundle = opencti_api_client.stix2.get_stix_bundle_or_object_from_entity_id(
-            entity_type="Malware", entity_id=self.entity_id, mode="full"
+            entity_type=self.entity_type, entity_id=self.entity_id, mode="full"
```

Comment:
This change is unrelated to the fix. It's just to make this script more usable in other conditions.
I have updated it during my dev, but I thought it was still something good to keep.